src/main/target/system_stm32f7xx.c

```diff
@@ -242,6 +242,20 @@
           while (1);
       }
 
+#ifdef USEMCO2ASOSDCLOCK
```

Comment:
Please use underscores to make this now readable, like it's done for every single `USE_` macro in Betaflight.